packages/astro/src/vite-plugin-env/index.ts

```diff
@@ -79,6 +79,7 @@ export default function envVitePlugin({
 				if (privateEnv) {
 					privateEnv.SITE = astroConfig.site ? `'${astroConfig.site}'` : 'undefined';
 					privateEnv.SSR = JSON.stringify(true);
+					privateEnv.STREAMING = JSON.stringify(astroConfig.server.streaming);
```

Comment:
do we need this? I think Matthew mentioned something internal that was already set to control this.

Reply:
Actually, no we don't!  `core/app/index.ts` needs to know whether streaming is enabled without access to the Astro config, I was using the env flag to pass along that information

This is a pretty specific use case though, probably better to just make the streaming option a param for `App`

Reply:
ah! yea, it's tempting to overload ENV for these things but lets avoid it when we can.